misc/services/camel/de-metas-camel-externalsystems/de-metas-camel-grssignum/src/main/java/de/metas/camel/externalsystems/grssignum/from_grs/vendor/processor/PushBPartnersProcessor.java

```diff
@@ -79,4 +81,14 @@ private BPUpsertCamelRequest getBPUpsertCamelRequest(@NonNull final JsonBPartner
 				.orgCode(credentials.getOrgCode())
 				.build();
 	}
+
+	@NonNull
+	private static String computeBPartnerIdentifier(@NonNull final JsonBPartner jsonBPartner)
+	{
+		if (jsonBPartner.getMetasfreshId() != null && Check.isNotBlank(jsonBPartner.getMetasfreshId()))
```

Comment:
```suggestion
		if (Check.isNotBlank(jsonBPartner.getMetasfreshId()))
```

again, wondering if "null" isn't already covered by isNotBlank

Reply:
yeah, it is, but Idea still pops a warning for it, so is just for ditching that warning